.github/workflows/build-java.yml

```diff
@@ -28,16 +28,30 @@ jobs:
         with:
           python-version: 3.9
       - name: Download workflow artifact
-        uses: dawidd6/action-download-artifact@v2.14.0
+        uses: dawidd6/action-download-artifact@v2.17.0
         with:
           workflow: "build-libs.yml"
           path: ./libs/${{ matrix.os-artifact[1] }}
           github_token: ${{ secrets.GITHUB_TOKEN }}
           name: ${{ matrix.os-artifact[1] }}
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - name: Set up Node (docs)
+        run: |
+          npm install groovydoc-to-markdown
+        shell: pwsh
+      - uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: '6.0.x'
+      - name: Set up Dotnet (docs)
+        run: |
+          dotnet tool install DefaultDocumentation.Console -g
+        shell: pwsh
```

Comment:
ah, I thought we might need to build dotnet docs with all of them since `build-dotnet` workflow doesn't call this `build_sdks` script.   but that's probably the real problem, we should make it call the script and allow for a real "build all" option